test/api/README.md

```diff
@@ -0,0 +1,21 @@
+# Testing the T-Route FastAPI Extension
+
+The following folder contains data files that are to be used to test the T-Route FastAPI code within src/app
+
+To use these files, follow the steps below:
+
+1. Copy the `test_compose.yaml` file in the root project dir 
```

Comment:
Can you be more specific with root dir? Do you mean in the `src` folder or lets say we are running LowerColorado in that folder

Reply:
Sure, I changed this to the following line:
```
Copy the `test_compose.yaml` file in the base project dir (/t-route)
```

Let me know if I need to be more specific

Reply:
Sure, I changed this to the following line:
```
Copy the `test_compose.yaml` file in the base project dir (/t-route)
```

Let me know if this should be more verbose.

_Edit: originally commented with my personal GitHub account. had to switch to my RTX one_

Reply:
The issue was that the `test_compose.yaml` file was in the wrong directory, but it's working now.

A couple more questions/comments:

When I run it, a browser opens to `http://localhost:8000/`, but I have to manually change it to `http://localhost:8004/docs` to display the correct page. Is there a way to open the correct page directly?

Could you provide an example of how to run the Lower Colorado test using `//t-route/test/LowerColorado_TX_v4/test_AnA_V4_HYFeature.yaml`?